packages/components/src/Disclosure/docs/customHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, Disclosure, DisclosurePanel, Div, Text } from "@hopper-ui/components";
+
+import { ToggleArrow } from "../../ToggleArrow/index.ts";
+
+export default function Example() {
+    return (
+        <Div width="100%" paddingX="core_320" paddingY="core_480">
+            <Disclosure>
+                {({ isExpanded }) => (
+                    <>
+                        <Button slot="trigger" variant="secondary">
+                            {({ isDisabled, isFocusVisible, isHovered, isPressed }) => (
+                                <>
+                                    <Text>Shipping & Returns Overview</Text>
+                                    <ToggleArrow
+                                        isExpanded={isExpanded}
+                                        isDisabled={isDisabled}
+                                        isFocused={isFocusVisible}
+                                        isHovered={isHovered}
+                                        isPressed={isPressed}
+                                        order="2"
```

Comment:
why is the order="2" required. ToggleArrow is an icon, and should be placed at the right place automatically no?

Reply:
This is a custom header, which means the user needs to style it. We do not have any default styles if they don't use our DisclosureHeader. Sure I could put order: 2; directly on the icon or something, but it may not be what they want.

Reply:
let's ignore the disclosure for now, if i do this: 

```
<Button slot="trigger" variant="secondary">
                                    <Text>Shipping & Returns Overview</Text>
                                    <ChevronIcon slot="end-icon" />
</Button>
```

This would render the icon after the text right, as seens components/Button#usage-end-icon example. Why aren't we doing this here? ToggleArrow is ChevronIcon

Reply:
oh wait.. this was stupid on my part. I think since it's an icon inside a Button, the button rendered it as the first element. I should be putting slot="end-icon"